pytripgui/canvas_vc/plotter/bars/bar_base.py

```diff
@@ -0,0 +1,33 @@
+from abc import ABC
+
+from matplotlib import pyplot as plt
+from matplotlib.axes import Axes
+from matplotlib.pyplot import colorbar
+
+from pytripgui.canvas_vc.plotter.bars.projection_enum import BarProjection
+
+
+# base abstract class for bars
+class BarBase(ABC, Axes):

```

Comment:
Add a comment describing responsibility of this class

Reply:
Done